components/swatch/index.css

```diff
@@ -283,6 +283,21 @@
 	}
 }
 
+/* the lightBorder does not change values between light/dark modes */
+.spectrum-Swatch--lightBorder {
+	.spectrum--light &,
+	.spectrum--dark &,
+	.spectrum--darkest & {
+		--spectrum-swatch-border-color: rgba(0, 0, 0, 20%);
+	}
```

Comment:
I'm impartial because I think this does the job... but because I am always curious about the historical context, I went digging and found [the PR](https://github.com/adobe/spectrum-css/pull/1501/files#diff-cc50ec1bfa218207591ac782352b4f5894b71d81450b2c74fa9004ca1dfca665) where `--lightBorder` disappears, and the CSS was originally like this:

```
.spectrum-Swatch--lightBorder {
  .spectrum-Swatch-fill {
    &:before {
      box-shadow: inset 0 0 0 var(--spectrum-swatch-fill-border-width) var(--spectrum-swatch-fill-border-color-light);
    }
  }
```
(note that we don't use `--spectrum-swatch-fill-border-width` anymore though)

And then the `--spectrum-swatch-fill-border-color-light` prop was defined ahead of that, kind of like how we usually define properties for hover, focus, etc.

Given how the code is in `main` now, it could fit up with the other placeholder token definitions something like:

```
	/* Light theme placeholder tokens */
	.spectrum--light & {
		--spectrum-swatch-border-color: rgba(0, 0, 0, 51%);
		--spectrum-swatch-fill-border-color-light: rgba(0, 0, 0, 20%);
	}

	/* Dark and Darkest theme placeholder tokens */
	.spectrum--dark &,
	.spectrum--darkest & {
		--spectrum-swatch-border-color: rgba(255, 255, 255, 51%);
		--spectrum-swatch-fill-border-color-light: rgba(0, 0, 0, 20%);
	}
```

Thoughts? Feel free to tell me I'm wrong, I'm not sure if one is better than the other! You've definitely spent more time looking at the swatch CSS than I have at this point!

Reply:
I don't know why I didn't think to put it at the top with the other light/dark/darkest variables. 🤦‍♀️ I like that organization though! https://github.com/adobe/spectrum-css/pull/2954/commits/ff94816cddad501e88340a04500d1e214cba86cf

As for the variable color change, you're definitely right. I misunderstood. I knew that token was new, but I didn't dig through the tokens dist. Now that I've re-read Miruna's response, the color will stay attached to `gray-1000`, but gray-1000 changes from light to dark. All I did was unattach gray-1000 for dark mode, which is wrong. Thanks for pointing that out!! https://github.com/adobe/spectrum-css/pull/2954/commits/f7559114fd4eefb4111e75c082054fa0d10a373e